test/models/search_facet_test.rb

```diff
@@ -0,0 +1,99 @@
+require "test_helper"
+
+class SearchFacetTest < ActiveSupport::TestCase
+
+  def setup
+    @search_facet = SearchFacet.first
+
+    # filter_results to return from mock call to BigQuery
+    @filter_results = [
+        {id: 'NCBITaxon_9606', name: 'Homo sapiens'},
+        {id: 'NCBITaxon_10090', name: 'Mus musculus'}
+    ]
+  end
+
+  # should return expected filters list
+  # mocks call to BigQuery to avoid unnecessary overhead
+  test 'should update filters list' do
+    puts "#{File.basename(__FILE__)}: #{self.method_name}"
+
+    # mock call to BQ, until a better library can be found
+    mock = Minitest::Mock.new
+    mock.expect :query, @filter_results, [String]
+
```

Comment:
oooh, that's slick how you made the bq_dataset method which made this test easier to write.  Nice!